src/devices/Common/Iot/Device/NumberTool.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device
```

Comment:
Can we move this into an additional namespace? Perhaps Iot.Device.Helpers or Iot.Device.Common? Currently Iot.Device doesn't contain any classes, only other namespaces, and I would like to keep it like that if possible.

Reply:
change it to `Iot.Device.Common`, because the class in the `Common` folder and I don't know if `Iot.Device.Helpers` should be `Helper` or `Helpers`😂